notebooks/requirements_dev.txt

```diff
@@ -17,16 +17,13 @@
 assertpy==1.1
 bump2version==1.0.1
 build==0.7.0
-flake8==4.0.1
-ipython
+ipython==7.32.0
```

Comment:
Not against this change, but can you move this and the change in requirements_lint.txt to another PR?